t/FHEM/00_SIGNALduino/01_SIGNALduino_Parse_MU.t

```diff
@@ -15,56 +15,63 @@ my @mockData = (
         plan => 2,
         testname =>  q[Corrupt Mu data, combined message],
         input =>  q[MU;P0=-2272;P1=228;P2=-356;P3=635;P4=-562;P5=433;D=012345234345252343452523434345252345234343434523434345252343452525252525234523452343452345252525;CP=5;R=4�;P3=;L=L=-2864;L=H=2980;S=L=-1444;S=H=1509;D=354146333737463037;C==1466;L==32;R==9;],
-        rValue => U(), 
-        todoReason => q[This data should not be processed]
+        rValue => U(),
     },
     {
         deviceName => q[dummyDuino],
         plan => 2,
         testname =>  q[Corrupt MU data, unknown specifier V=],
         input =>  q[MU;P0=-1440;P1=432;P2=-357;P3=635;P4=-559;D=012121212123412343412123434121234343412123412343434341234343412123434121212121212341231212343412341212121;CP=1;V=139;],
         rValue => U(), 
-        todoReason => q[This data should not be processed]
     },
     {
         deviceName => q[dummyDuino],
         plan => 2,
         testname =>  q[Corrupt MU data, missing D= part],
-        input =>  q[MU;P0=-370;P1=632;P2=112;P3=-555;P4=428;P5=-780;P6=180;P7=;CP=4;R=77;],
+        input =>  q[MU;P0=-370;P1=632;P2=112;P3=-555;P4=428;P5=-780;P6=180;P7=-200;CP=4;R=77;],
+        rValue => U(), 
+    },
+    {
+        deviceName => q[dummyDuino],
+        plan => 2,
+        testname =>  q[Corrupt MU data, D= is to short (min two pattern)],
+        input =>  q[MU;P0=-370;P1=632;P2=112;P3=-555;P4=428;P5=-780;P6=180;P7=-200;D=1;CP=4;R=77;],
         rValue => U(), 
-        todoReason => q[This data should not be processed]
     },
     {
         deviceName => q[dummyDuino],
         plan => 2,
         testname =>  q[Corrupt MU data, C= has letters ],
         input =>  q[MU;P0=536;P1=-1443;P2=1486;P3=-4208;P4=5776;P5=-6700;P6=2972;P7=-2880;D=01212121212121212123456767212121212121672167212121212121212121672167672;C=23RB;],
         rValue => U(), 
-        todoReason => q[This data should not be processed]
     },
     {
         deviceName => q[dummyDuino],
         plan => 2,
         testname =>  q[Corrupt MU data, wrong delemiter =],
         input =>  q[MU;P0=536;P1=-1443;P2=1486;P3=-4208;P4=5776;P5=-6700;P6=2972;P7=-2880;D=01212121212121212123456767212121212121672167212121212121212121672167672=C=23;],
         rValue => U(), 
-        todoReason => q[This data should not be processed]
     },
     {
         deviceName => q[dummyDuino],
         plan => 2,
         testname =>  q[Corrupt MU data, D= isn't numeric],
         input =>  q[MU;P0=-16320;P1=394;P2=-400;P5=628;P6=-625;D=0121212121212121212121212561A1256121212121256121256125656C212561256561256121256121256565612565656121212125612125612121256121256125612561;CP=1;R=84;],
```

Comment:
@HomeAutoUser

Das hier müsste die 2. rmsg sein, welche bisher dazu führte das parse in FS10 aufgerufen wurde

Reply:
Es ist natürlich eine plausible Erläuterung, das eine bisherige "Nichtprüfung" der Datenteile zu einem Dispatch in der FS10 Parse führte.